tests/telemetry/test_telemetry.py

```diff
@@ -313,7 +307,8 @@ def my_function(payload):
 
     my_function()
 
-    call2_kwargs = mock_posthog_capture.call_args_list[1][1]
+    print(mock_posthog_capture.call_args_list)
```

Comment:
looks like you left a debug statement here. otherwise, lgtm!

Reply:
whops! yes, you are right!